.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Add a brief summary of your PR here, between 1 sentence and two paragraphs
```

Comment:
Need to spell out "one", here. The really formal rule is to spell out numbers 10 or less. Alternatively, use the numeric "2". I would go with the former.